Shared/Samples/Set feature layer rendering mode on scene/SetFeatureLayerRenderingModeOnSceneView.swift

```diff
@@ -0,0 +1,166 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SetFeatureLayerRenderingModeOnSceneView: View {
+    /// Scene that displays with dynamic rendering.
+    @State private var dynamicScene = Scene()
+    
+    /// Scene that displays with static rendering.
+    @State private var staticScene = Scene()
+    
+    /// A Boolean value indicating whether the scene views are currently zooming.
+    @State private var isZooming = false
+    
+    /// The viewpoint for the scene.
+    @State private var viewpoint: Viewpoint?
+    
+    /// A Boolean value indicating whether the scene is fully zoomed in.
+    @State private var isZoomedIn = true
+    
+    /// Creates service feature tables using point, polygon, and polyline services.
+    let featureTables: [ServiceFeatureTable] = [URL.pointTable, .polylineTable, .polygonTable].map(ServiceFeatureTable.init(url:))
+    
+    init() {
+        // Iterate through the feature tables and use them to set up feature layers.
+        // Set the rendering mode for either dynamic or static rendering,
+        // and add the feature layers to the scene.
+        for featureTable in featureTables {
+            // Set up the dynamic scene first.
+            let dynamicFeatureLayer = FeatureLayer(featureTable: featureTable)
+            dynamicFeatureLayer.renderingMode = .dynamic
+            dynamicScene.addOperationalLayer(dynamicFeatureLayer)
+            // Then set up the static scene using a clone of the dynamic feature layer.
+            let staticFeatureLayer = dynamicFeatureLayer.clone()
+            staticFeatureLayer.renderingMode = .static
+            staticScene.addOperationalLayer(staticFeatureLayer)
+        }
+        staticScene.initialViewpoint = .zoomedOutViewpoint
+        dynamicScene.initialViewpoint = .zoomedOutViewpoint
+    }
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            SceneViewReader { sceneViewProxy in
+                SceneView(scene: staticScene)
```

Comment:
I suggest to follow the [iOS implementation](https://github.com/Esri/arcgis-runtime-samples-ios/blob/main/arcgis-ios-sdk-samples/Features/Feature%20layer%20rendering%20mode%20(scene)/FeatureLayerRenderingModeSceneViewController.swift#L43) to disable interactions on the scene views with `.interactiveNavigationDisabled(true)` modifier.